chains/razorSChain/validatorList.json

```diff
@@ -60,6 +60,12 @@
 	    "name": "StakeShack",
 	    "Website": "",
 	    "logo": "shacklogo.jpg"
-    	}
+    	},
+	"27": {
+	    "address": "0x9f13929467EE138d465bb5dAb717693637B68656",
+	    "name": "ByteVariety",
+	    "Website": "https://bytevariety.com/",
+	    "logo": "bytevariety.jpg"
```

Comment:
@alByteVariety please fix the logos extension, ie `jpeg`

Reply:
@yohanelly95 Changed as advice, thanks.